components/avatar/style/index.less

```diff
@@ -32,6 +32,11 @@
     border-radius: @avatar-border-radius;
   }
 
+  &-cushion {
+    mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 200 200'%3E%3Cpath d='M100,0c78,0,100,22,100,100S178,200,100,200,0,178,0,100,22,0,100,0Z'/%3E%3C/svg%3E");
```

Comment:
`mask-image` is not compatible with IE.

https://developer.mozilla.org/en-US/docs/Web/CSS/mask-image